config/environment.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function(environment, ENV) {
+  ENV.APP = ENV.APP || {};
+  ENV.APP.options = ENV.APP.options || {};
+  ENV.APP.options['ember-cli-qunit'] = ENV.APP.options['ember-cli-qunit'] || {};
+
+  if (environment === 'test' && ENV.APP.options['ember-cli-qunit'].rootElement) {
+    ENV.APP.rootElement = ENV.APP.options['ember-cli-qunit'].rootElement;
+  } else {
+    ENV.APP.rootElement = '#ember-testing';
```

Comment:
Unrelated to the other conversation, I do not think that mutating `ENV.APP.rootElement` in an addon should propogate to the app's final configuration.  We do not do a deep merge, so the fact that `ENV.APP` is already present on `ENV` ([here](https://github.com/ember-cli/ember-cli/blob/fba306822da476f4debd2173e83fd925a286727b/blueprints/app/files/config/environment.js#L16-L19)) will mean that this setting is completely clobbered and lost from the final config in the app.

Background: 

* https://github.com/ember-cli/ember-cli/blob/fba306822da476f4debd2173e83fd925a286727b/lib/broccoli/broccoli-config-loader.js#L53
* https://github.com/ember-cli/ember-cli/blob/fba306822da476f4debd2173e83fd925a286727b/lib/models/project.js#L199-L202
* https://lodash.com/docs#merge